content/docs/start/data-and-model-versioning.md

```diff
@@ -174,9 +171,9 @@ $ rm -f data/data.xml
 </tab>
 <tab title="Windows">
 
-```dvc
-$ rmdir .dvc\cache
-$ del data\data.xml
+```powershell
```

Comment:
👍 let's create though a ticket to do a proper `dvc` highlighter that supports windows

Reply:
https://github.com/iterative/dvc.org/issues/3278